test/misc-components/downloadItemTest.js

```diff
@@ -31,6 +31,8 @@ describe('downloadItem test', function () {
     yield setup(this.app.client)
 
     yield this.app.client
+      .changeSetting('general.download-always-ask', false)
```

Comment:
Only later I notice that this can be variable, so I change it for 0.21 and 0.22

https://github.com/brave/browser-laptop/blob/e6225e0a6acf67eaf40dc362f989d17010cc4cc2/test/misc-components/downloadItemTest.js#L88-L89